ui/apps/ui/src/app/components/results-with-pagination/result.component.html

```diff
@@ -1,6 +1,33 @@
-<div class="type-icon-holder {{ type.label }}">
-  <div class="type-icon"></div>
-</div>
+<ng-container
+  *ngIf="
+    type.value === 'service' ||
+      type.value === 'data-source' ||
+      type.value === 'provider' ||
+      type.value === 'data source';
```

Comment:
Why `data-source` and `data source`?

Reply:
By the way, offers in bundles also should have icons, right? @agpul

Reply:
old code, its just a reformat :)